packages/components/src/lib/controller/overlay/OverlayController.ts

```diff
@@ -10,8 +10,8 @@ export interface OverlayControllerOptions {
 
 export class OverlayController {
```

Comment:
Maybe you should introduce two new local types for the handler and disposer to clarify the intent of this functions.

```typescript
type OpenStateHandler = () => void;
type DisposerFn = () => void;
```